starport/templates/typed/stargate/component/x/{{moduleName}}/keeper/{{typeName}}.go.plush

```diff
@@ -78,12 +78,12 @@ func (k Keeper) Has<%= TypeName.UpperCamel %>(ctx sdk.Context, id uint64) bool {
 	store :=  prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.<%= TypeName.UpperCamel %>Key))
 	return store.Has(Get<%= TypeName.UpperCamel %>IDBytes(id))
 }
-
+<%= if (!NoMessage) { %>
```

Comment:
I think we can even totally remove this method. This one is even a bit dirty since it doesn't check if the type exist.

We could handle it in a separate PR where we remove the `Has...` method and this one to keep a single `Get` that returns the type and a boolean, so that we keep the package consistent and simple

Reply:
Totally agree. I will create a new PR for it. and also complement this infos into issue #1409

Reply:
Created an issue a well. In fact, it's still better when we can see assignee